students/mattclau/lesson09/donor_models.py

```diff
@@ -0,0 +1,109 @@
+from operator import itemgetter
+import os
+
+class Donor:
+
+    def __init__(self, name, donations = [None]):
+        """creates a Donor object"""
+        self.name = name
+        self.donations = []
+        if donations != [None]:
+            for donation in donations:
+                self.donations.append(donation)
+
+
+    def create_letter(self,donor):
```

Comment:
this method should use `self` to get the donor - no need to pass in donor

for example `self.name` instead of `donor.name`